packages/core/chrome/core-chrome-browser-internal/src/ui/header/collapsible_nav.scss

```
@@ -24,3 +24,19 @@
     text-decoration: underline; /* 2 */
   }
 }
+
+/**
+ * Allows the CollapsibleNavGroups to be viewed on
+ * very small screen sizes and when the browser Zoom is high
+ */
+@media (max-height: 15em) {
+  .euiCollapsibleNav {
+    overflow-y: auto;
+
+    & > .euiFlexItem.eui-yScroll {
+      flex-shrink: 0;
+      flex-grow: 1;
+      flex-basis: auto;
+    }
```

Similarly, we should also be setting a targeted className (e.g. `.kbnCollapsibleNav__solutions`) on the specific EuiFlexItem listed, instead of targeting an arbitrary class. We can then remove the `eui-yScroll` util and use a Sass util instead.
```suggestion
.kbnCollapsibleNav__solutions {
  @include euiYScroll;
  flex: 1 0 auto;
}
```